src/app/components/sidebar/sidebar.component.ts

```diff
@@ -24,7 +26,9 @@ export class SidebarComponent implements OnInit{
   screenWidth = 0 ;
   navData=sidebarData;
   user?:User;
-  authService:AuthenticationService = inject(AuthenticationService);
+  userInfo:any;
```

Comment:
ur using this userinfo everywhere, define a model it would be easier to understand which data  are in it